src/java.base/share/classes/java/util/Optional.java

```diff
@@ -71,8 +75,9 @@
 "meaning, but are unrelated by the Java type hierarchy.",
 "@Covariant makes Optional<@NonNull String> a subtype of Optional<@Nullable String>."
 })
-@AnnotatedFor({"lock", "nullness"})
-public final class Optional<T> {
+@AnnotatedFor({"lock", "nullness", "optional"})
```

Comment:
Still missing nullness annotations in this file.

Reply:
@smillst Please merge #8 before merging this.
Thanks.